### ISSUE TITLE
TypeError: 'float' object is not subscriptable

### ISSUE THREAD

**Issue by dxcore35** [g-battaglia/kerykeion#2]
**System:** Mac OS. Python 3.7
**Installation:**` pip3 install Kerykeion`

```
/usr/local/lib/python3.7/site-packages/kerykeion/astrocore.py in planets_lister(self)
    221 
    222         """Calculates the position of the planets and stores it in a list."""
--> 223         self.sun_deg = swe.calc(self.j_day, 0, self.iflag)[0][0]
    224         self.moon_deg = swe.calc(self.j_day, 1, self.iflag)[0][0]
    225         self.mercury_deg = swe.calc(self.j_day, 2, self.iflag)[0][0]

TypeError: 'float' object is not subscriptable
```

**Comment by g-battaglia** [comment]
Should be fixed in version 0.0.2.2! I updated the it on PyPi later, try to download it again now

**Comment by dxcore35** [comment]
It works now! I thought the pip is automatically taking the latest source code from github repo tho.
